src/kaleidoscope_internal/deprecations.h

```diff
@@ -46,3 +46,10 @@
   "Layers are now in activation-order, please use" __NL__ \
   "`Layer.forEachActiveLayer()` instead."
 
+#define _DEPRECATED_MESSAGE_LAYER_UPDATELIVECOMPOSITEKEYMAP             __NL__ \
```

Comment:
I'm not sure this is a public enough API to -need- the deprecation song and dance. @algernon - what do you think?

Reply:
I'd be a bit surprised if `Layer.eventHandler()` gets called from anyplace else. On the other hand, it can be deprecated and removed along with `updateLiveCompositeKeymap()`, so the cost is minimal. If the call were mine to make, I don't know which option I'd pick.

Reply:
More definitive: `Layer.eventHandler()` does not get called by any existing core plugin. It probably doesn't get called by any third-party plugins, but that's not impossible.